wordpress-mobile/WordPress-iOS
# Jetpack Prompt:  Add logic to dismiss the prompt if the user selects no thanks

Project: #17721 

Note: I committed to the wrong branch and reverting wasn't working right so these are the only changes I made:

https://github.com/wordpress-mobile/WordPress-iOS/pull/18319/files/3ed06452bed7bcf4978588c47ee6f5a73887ab8d..c1d81d74ff51d0f6d898ea2a99e540afdf30c2b7

### Description
When tapping the Install button on the Jetpack Install Prompt you are brought to the Jetpack install process

**Notes:** 
- It is expected that the UI tests are broken for this PR. Testing will be addressed in a later PR.


### Video

https://user-images.githubusercontent.com/793774/162999315-c3424234-bea9-4781-a401-7e6dbbea979c.mp4


### To test:

1. Launch the app
2. Logout
3. Add a self hosted site that is not connected to Jetpack
4. Verify you are shown the prompt view as displayed above
5. Tap the No thanks button
6. You should be brought to the home screen
7. Sign out / delete the self hosted site to be brought back to the login view
8. Add  the same self hosted site again
9. Verify you are not shown the prompt again


### Regression Notes
1. Potential unintended areas of impact
None. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None yet. 

### PR submission checklist:
